runtime/src/bank.rs

```diff
@@ -1196,7 +1255,7 @@ impl Bank {
                     .map(|drop_callback| drop_callback.clone_box()),
             )),
             freeze_started: AtomicBool::new(false),
-            cost_tracker: RwLock::new(CostTracker::default()),
+            cost_tracker: RwLock::new(parent.read_cost_tracker().unwrap().new_from_parent()),
```

Comment:
@tao-stones i think this is uncontroversial change and there should be no risk of different behavior without feature flag.

Reply:
Yea I think so. Tho the name `new_from_parent` made me guess if it carries accumulated costs from parent. Perhaps `new_with_parent_limits`, wdyt?

Reply:
thanks for nice naming suggestion!: 7ff8cbc3306f0e05a3df99d198c980acc5dcaf0a